_locales/_untranslated_en.json

```diff
@@ -23,5 +23,8 @@
   },
   "pref_current_account": {
     "message": "Current Account"
+  },
+  "camera_access_failed": {
+    "message": "Could not access video camera"
```

Comment:
maybe add a hint about permissions though I don't know how common real errors vs permission errors are, so maybe not.

Reply:
I've added some more advice :+1: